rspace/src/main/scala/coop/rchain/rspace/channelStore/syntax/ChannelStoreSyntax.scala

```diff
@@ -0,0 +1,80 @@
+package coop.rchain.rspace.channelStore.syntax
+
+import cats.effect.Concurrent
+import cats.syntax.all._
+import coop.rchain.rspace.Blake2b256Hash
+import coop.rchain.rspace.channelStore.{ChannelStore, ContinuationHash, DataJoinHash}
+import coop.rchain.rspace.trace.{Consume, Produce}
+import coop.rchain.shared.syntax._
+
+trait ChannelStoreSyntax {
+  implicit final def syntaxChannelStore[F[_], C](
+      channelStore: ChannelStore[F, C]
+  ): ChannelStoreOps[F, C] =
+    new ChannelStoreOps[F, C](channelStore)
+}
+
+final case class DatumMapping(historyHash: Blake2b256Hash, eventLogHash: Blake2b256Hash)
+final case class ConsumeMapping(historyHash: Blake2b256Hash, eventLogHashes: Seq[Blake2b256Hash])
+final case class JoinMapping(joinHash: Blake2b256Hash, eventLogHashe: Blake2b256Hash)
```

Comment:
typo. `eventLogHashe`

Reply:
Another question on this JoinMapping. I don't see this is in use anywhere. I think that is because we do nothing on join now, right?

Reply:
@zsluedem nice catch! Logic of merger assumes that joins hashes are read on-demand with LazyCache
https://github.com/rchain/rchain/blob/79d42ba36857599a2fcc75ac8af18ec5b3e71184/rspace/src/main/scala/coop/rchain/rspace/merger/instances/DiffStateMerger.scala#L73-L87
 because joins changes computation depends on consume changes.
https://github.com/rchain/rchain/blob/79d42ba36857599a2fcc75ac8af18ec5b3e71184/rspace/src/main/scala/coop/rchain/rspace/merger/instances/DiffStateMerger.scala#L192-L197

So we preload only datums and konts changes, therefore `JoinMapping` is not used. But might be later it will find a use. 
https://github.com/rchain/rchain/blob/79d42ba36857599a2fcc75ac8af18ec5b3e71184/rspace/src/main/scala/coop/rchain/rspace/merger/instances/DiffStateMerger.scala#L127-L128

Reply:
Please comment out the code if it's not used and why is left.